src/ui/screens/onboarding/OnboardingStartSlideDesktop.qml

```diff
@@ -34,40 +34,52 @@ ColumnLayout {
 
     MZInterLabel {
         Layout.topMargin: 8
-        Layout.leftMargin: MZTheme.theme.windowMargin * 2
-        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.leftMargin: MZTheme.theme.windowMargin * 1.5
+        Layout.rightMargin: MZTheme.theme.windowMargin * 1.5
         Layout.fillWidth: true
 
         text: MZI18n.OnboardingStartSlideBody
         horizontalAlignment: Text.AlignLeft
         color: MZTheme.theme.fontColor
     }
 
-    MZCheckBoxRow {
-        objectName: "startAtBootCheckBox"
-
-        Layout.topMargin: MZTheme.theme.vSpacing
-        Layout.leftMargin: MZTheme.theme.windowMargin * 2
-        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+    RowLayout {
+        Layout.topMargin: MZTheme.theme.vSpacingSmall
+        Layout.leftMargin: MZTheme.theme.windowMargin * 1.5
+        Layout.rightMargin: MZTheme.theme.windowMargin * 1.5
         Layout.fillWidth: true
 
-        subLabelText: MZI18n.SettingsStartAtBootTitle
-        leftMargin: 0
-        isChecked: MZSettings.startAtBoot
-        showDivider: false
-        onClicked: {
-            MZSettings.startAtBoot = !MZSettings.startAtBoot
+        MZBoldInterLabel {
+            id: connectOnStartupLabel
+            Layout.fillWidth: true
 
-            if (MZSettings.startAtBoot) {
-                Glean.interaction.connectOnStartupEnabled.record({
-                    screen: root.telemetryScreenId,
-                });
-            }
-            else {
-                Glean.interaction.connectOnStartupDisabled.record({
-                    screen: root.telemetryScreenId,
-                });
+            text: MZI18n.SettingsStartAtBootTitle
+            font.pixelSize: MZTheme.theme.fontSize
+            lineHeight: MZTheme.theme.labelLineHeight
+        }
+
+        MZSettingsToggle {
+            objectName: "startAtBootToggle"
+
+            implicitHeight: 24
```

Comment:
Same theme vars comment as before. Also, do we expect the sizing here to ever change? Could we specify this in MZSettingsToggle and then just overwrite it when necessary?